Remotion/Web/Core/UI/Controls/WebTreeView.cs

```diff
@@ -1388,6 +1424,13 @@ protected virtual string CssClassNodeChildrenNoLines
       get { return "treeViewNodeChildrenNoLines"; }
     }
 
+    /// <summary> Gets the CSS-Class applied to the <see cref="WebTreeView"/>'s last node in a category. </summary>
+    /// <remarks> Class: <c>CssClassNodeSeparator</c> </remarks>
```

Comment:
Maybe we should make the name of the css variable more specific? `treeViewNodeCategorySeparator`? What do you think?

```suggestion
    /// <remarks> Class: <c>treeViewSeparator</c> </remarks>
```

Reply:
yeah sounds like a good idea